cmd/artesia/app/config.go

```diff
@@ -1,13 +1,52 @@
 package app
 
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/viper"
+)
+
 // Config contains config data for the server
 type Config struct {
 	ListenAddress string
+	Database      DatabaseConfig
+}
+
+// DatabaseConfig has the data needed to connect to a Postgres DB
+type DatabaseConfig struct {
+	Host     string
+	DBName   string
+	User     string
+	Password string
+	Port     int
 }
 
-// NewDefaultConfig returns a default config
-func NewDefaultConfig() *Config {
-	return &Config{
-		ListenAddress: "127.0.0.1:8080",
+// LoadConfig tries to load a config file from the ARTESIA_CONFIG_FILE environment variable. It should be an absolute path
+func LoadConfig() (*Config, error) {
+	v := viper.New()
+	configFilePath, err := filepath.Abs(os.Getenv("ARTESIA_CONFIG_FILE"))
+	if err != nil {
```

Comment:
If we're going to quit on a missing config file, then let's put this at the beginning of `LoadConfig` so we can escape early.